Dnn.AdminExperience/ClientSide/Servers.Web/src/components/Tabs/Performance.jsx

```diff
@@ -70,7 +70,7 @@ class Performance extends Component {
 
     onChangeField(key, event) {
         let value = event;
-        if (event && event.value) {
+        if (event && event.value !== undefined) {
             value = event.value;
         } else if (event && event.target && event.target.value) {
```

Comment:
Do we know the scenario when this `if` fires? Should we also be doing `!== undefined` for `event.target.value`?

Reply:
@bdukes `onChangeField()` is bound to multiple inputs on Performance.jsx. I triggered value change for all the inputs on the page. There was no any case when ` if (event && event.target && event.target.value) `got fired.

Reply:
My guess is that it is for browser compatibility 🤷 if you could also add that condition in the elseif it would be nice, if not then I guess it is fine for browsers most admins use.

Reply:
@valadas Please let me know in case changes of "else if" are blocking the merge.

Reply:
@manojattal please add the same check to the `else if` branch. Thanks!

Reply:
@bdukes I have added the check. Please let me know if anything required. Thanks!